safe/impact_functions/inundation/tsunami_raster_road/impact_function.py

```diff
@@ -154,6 +167,9 @@ def _raster_to_vector_cells(raster, ranges, output_crs):
                 features = []
     # Add the latest features
     vl.dataProvider().addFeatures(features)
+    # For debugging
+    # from qgis.core import QgsMapLayerRegistry
+    # QgsMapLayerRegistry.instance().addMapLayer(vl)
```

Comment:
These lines can be rewritten easily. If we let these kind of lines, we will find a lot in a few months/years.